.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Continuous Integration
+concurrency: 
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - name: Set up JDKs ☕
+      uses: actions/setup-java@v3
+      with:
+        java-version: |
+          8
+          17
+          20
+        mvn-toolchain-id: |
+          JavaSE-1.8
```

Comment:
Here also Java 8?

Reply:
No, this is a specific Tycho/OSGi label here: tycho wants a toolchain that has the BREE name as id to properly resolve it, and for Java 8, it's JavaSE-1.8. See https://github.com/eclipse-tycho/tycho/issues/2670 .
Note that this particular workaround is necessary for the older org.eclipse.jdt.annotations bundle only.